tt_metal/llrt/tt_cluster.hpp

```diff
@@ -91,6 +91,8 @@ class Cluster {
         void *mem_ptr, uint32_t sz_in_bytes, tt_cxy_pair core, uint64_t addr, bool small_access = false) const;
     void read_core(
         std::vector<uint32_t> &data, uint32_t sz_in_bytes, tt_cxy_pair core, uint64_t addr, bool small_access = false) const;
+    void read_core(
```

Comment:
same thing here, don't think this is needed since there are read_core variants that accept void*